src/test_c2s_srv.c

```diff
@@ -472,10 +472,9 @@ int test_c2s(Connection *ctl, tcp_stat_agent *agent, TestOptions *testOptions,
   create_client_logdir((struct sockaddr *) &cli_addr[0], clilen,
                        options->c2s_logname, sizeof(options->c2s_logname),
                        namesuffix, sizeof(namesuffix));
-  sleep(RACE_CONDITION_WAIT_TIME);
+  mysleep(RACE_CONDITION_WAIT_TIME);
```

Comment:
The problems with `alarm()` and `sleep()` are twofold:
  1) `sleep()` may cause an `alarm()`'s `SIGALRM` to be missed. 
  2) `sleep()` may reset the `alarm()` timer. 
Neither case is a problem for us as long as we set an alarm immediately after the call to sleep. If you still aren't comfortable with this usage, then please use `nanosleep()`, which is safe to use with `alarm()`.  `mysleep()` is not the thing to use.